build/infrastructure/eo/chart/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: eo-frontend
 description: A Helm chart for Kubernetes
 
 type: application
-version: 0.2.25
+version: 0.2.26
```

Comment:
Please revert this, updating EO should only be necessary when changing in shared functionality across datahub and energioprindelse like the design system (watt)

Reply:
It was me who asked Karsten to bump the version because the pipeline was failing otherwise. So for some reason the DH's API also affects EO.

Reply:
These changes shouldn't affect EO. I just checked the dep-graph to make sure that isn't the problem, but the only code they have in common is gf-test-util-matchers. However I can see that the affected check did error:

![image](https://user-images.githubusercontent.com/86852536/164163014-03286713-dab2-402d-b905-8a12b0da2373.png)

I would like to revert the files, and try running the check again

Reply:
Hmm somethings wrong, it triggers a build of some weird reasons. I increase the versions again and then we must talk with @LayZeeDK about this.

Reply:
When I run `nx affected --target=test --skip-nx-cache` I get the following list:

dh-shared-data-access-api
  - dh-core-shell
  - app-dh
  - dh-shared-domain
  - dh-metering-point-feature-identity-and-master-data
  - dh-metering-point-feature-overview
  - dh-metering-point-shell
  - dh-market-participant-data-access-api
  - dh-market-participant-feature-organization
  - dh-market-participant-shell
  - dh-message-archive-feature-log-search
  - dh-message-archive-shell
  - dh-message-archive-data-access-api
  - dh-metering-point-data-access-api
  - dh-metering-point-feature-search
  - dh-metering-point-feature-tabs
  - dh-metering-point-ui-status-badge
  - dh-metering-point-shared-ui-util
  - dh-charges-data-access-api
  - dh-shared-data-access-msw
  - dh-shared-test-util-msw
  - dh-shared-environments
  - dh-globalization-configuration-localization
  - dh-shared-test-util-i18n
  - ui-watt
  - **eo-auth-feature-terms**
  - **eo-core-shell**
  - **app-eo**
  - **eo-landing-page-shell**
  - **eo-dashboard-shell**
  - dh-auth-feature-msal
  - dh-shared-test-util-auth
  - dh-metering-point-domain
  - dh-metering-point-routing
  - gf-test-util-staging
  - dh-globalization-data-access-localization
  - **eo-shared-atomic-design-feature-molecules**
  - **eo-privacy-policy-shell**
  - dh-globalization-feature-language-picker
  - **eo-shared-atomic-design-ui-organisms**
  - dh-shared-test-util-metering-point
  - **eo-shared-atomic-design-ui-atoms**
  - **eo-origin-of-energy-shell**
  - **eo-auth-data-access-security**
  - **eo-origin-of-energy-routing**
  - **eo-metering-points-routing**
  - dh-message-archive-domain
  - **eo-privacy-policy-routing**
  - **eo-auth-routing-security**
  - **eo-metering-points-shell**
  - dh-globalization-domain
  - **eo-auth-data-access-api**
  - **eo-landing-page-routing**
  - dh-shared-ui-date-time
  - **eo-shared-environments**
  - **eo-shared-util-browser**
  - **eo-dashboard-routing**
  - **eo-faq-routing**
  - **eo-faq-shell**
  - gf-test-util-matchers
  - gf-util-browser
  - workspace-tools